plugins/microsoft-calendar/src/components/CalendarCard.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { sortBy } from 'lodash';
+import { DateTime } from 'luxon';
+import React, { useState } from 'react';
+
+import { InfoCard, Progress } from '@backstage/core-components';
+// import { useAnalytics } from '@backstage/core-plugin-api';
+
+import { Box, IconButton, Typography } from '@material-ui/core';
+import PrevIcon from '@material-ui/icons/NavigateBefore';
+import NextIcon from '@material-ui/icons/NavigateNext';
+
+import { useCalendarsQuery, useEventsQuery, useSignIn } from '../hooks';
+import microsoftCalendarIcon from '../icons/microsoft-outlook.svg';
+import { CalendarEvent } from './CalendarEvent';
+import { CalendarSelect } from './CalendarSelect';
+import { SignInContent } from './SignInContent';
+import { getStartDate } from './util';
+import useAsync from 'react-use/lib/useAsync';
+
+export const CalendarCard = () => {
+  const [date, setDate] = useState(DateTime.now());
+  // const analytics = useAnalytics();
+  const [selectedCalendarId, setSelectedCalendarId] = useState('');
+
+  const changeDay = (offset = 1) => {
+    setDate(prev => prev.plus({ day: offset }));
+    // analytics.captureEvent('click', 'change date');
```

Comment:
Can you remove the commented out code in this file if it's not going to be used?